ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -1710,24 +1686,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-proxy-workers-rhcos-rhel8-f14
```

Comment:
I guess we need change the job to `azure-ipi-proxy` so that cover proxy testing.

Reply:
@jianlinliu I see we still have a job covering this as
`openshift-openshift-tests-private-release-4.19__multi-nightly.yaml:- as: azure-ipi-proxy-tp-arm-f7`

Reply:
I guess that one is for `tp` coverage.

Reply:
OK, updated this one to `azure-ipi-proxy`, and considering there's already another tp job testing on the proxy scenario, so keeping the original f14/f28 frequency.